### ISSUE TITLE
HackerEarth Holiday Session Challenge

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#63]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : HackerEarth Holiday Session Challenge</br>
:red_circle: **Aim** : Your task is to create a solution using deep learning to discern whether a post is holiday-related in an effort to better monetize the platform.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/oossiiris/hackerearth-deep-learning-challenge-holidayseason</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Vikalp981** [comment]
Full name: Vikalp Kumar Tripathi
GitHub Profile Link: https://github.com/Vikalp981
Email ID: tripathivikalp862@gmail.com
Approach for this Project: First, I am thinking of using K-fold and then comparing the accuracy by implying other transfer learning algorithms with the help of TensorFlow
What is your participant role? SSOC

**Comment by abhisheks008** [comment]
@Vikalp981 updates please.

**Comment by Vikalp981** [comment]
Sir please unassign this issue to me as I have tried implementing it but facing errors which my laptop can not handle. So I need to leave this. Sorry from my side

**Comment by abhisheks008** [comment]
Anyways, thanks for trying it out.

**Comment by kshitij192** [comment]
Full name : Kshitij Kumar
GitHub Profile Link : https://github.com/kshitij192
Email ID : kshitijayushk192@gmail.com
Approach for this Project : Using Inceptionresnet pretrained model by transfer learning
What is your participant role? SSOC

**Comment by abhisheks008** [comment]
Issue assigned to you @kshitij192
